tests/testthat/test-model_comparison_table.R

```diff
@@ -0,0 +1,69 @@
+library(dplyr)
+library(testthat)
+library(caret)
+library(mlbench)
+library(stats)
+library(e1071)
+library(caTools)
+library(tibble)
+library(purrr)
```

Comment:
like fanli say, probably try using `use_package()` to 'load' these packages ? instead of calling it here? remember to run check() afterwards.